src/components/molecules/tabs/request/response/ResponseBody.tsx

```diff
@@ -1,11 +1,11 @@
 import { Stack, useColorScheme } from '@mui/joy';
 import { Editor, Monaco } from '@monaco-editor/react';
 import { useEffect, useRef, useState } from 'react';
-import { log } from '../../../../utils/logging';
-import { FormatIcon } from '../../../atoms/buttons/FormatIcon';
-import { EndpointResponse } from '../../../../types/application-data/application-data';
-import { defaultEditorOptions } from '../../../../managers/MonacoInitManager';
-import { CopyToClipboardButton } from '../../../atoms/buttons/CopyToClipboardButton';
+import { log } from '../../../../../utils/logging';
```

Comment:
Big if aliased. Have you considered aliasing like we did before?

Reply:
big agree. Seems like a separate PR though tbh